airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -57,9 +59,10 @@
     SNOWFLAKE
   }
 
-  public DefaultNormalizationRunner(final DestinationType destinationType, final ProcessBuilderFactory pbf) {
+  public DefaultNormalizationRunner(final DestinationType destinationType, final ProcessBuilderFactory pbf, boolean useDevVersion) {
```

Comment:
It's open for discussion, I am thinking we can keep it...

The intended behavior is:
- to automatically use `normalization:dev` images when the destination is used with `dev`.
- If the destination is using a numbered version, then normalization is also using the declared numbered version.

It's just to avoid switching between number/dev image tags when working on a PR of normalization
(I noticed @sherifnada does that often too) and keep in mind to switch it back before merging

WDYT?

Reply:
cool. let's work through this feature in a separate PR.

Reply:
https://github.com/airbytehq/airbyte/issues/2054